cast/tests/utils/prepare_for_tests.sh

```diff
@@ -5,20 +5,32 @@ if ! which starknet-devnet > /dev/null 2>&1; then
   exit 1
 fi
 
-COMPILER_DIRECTORY="$(git rev-parse --show-toplevel)/cast/tests/utils/"
+UTILS_DIRECTORY="$(git rev-parse --show-toplevel)/cast/tests/utils/"
```

Comment:
it seems thid directory is no longer needed for other things than compiler? so maybe we can revert this change altogether?